src/tiles/clickable-tile.component.ts

```diff
@@ -0,0 +1,63 @@
+import {
+	Component,
+	Input
+} from "@angular/core";
+
+/**
+ * Build application's clickable tiles using this component.
+ *
+ * ## Basic usage
+ *
+ * ```html
+ * <ibm-clickable-tile>
+ * 		tile content
+ * </ibm-clickable-tile>
+ * ```
+ *
+ * @export
+ * @class ClickableTile
+ * @implements {OnInit}
+ */
+@Component({
+	selector: "ibm-clickable-tile",
+	template: `
+	<a
+		class="bx--link bx--tile bx--tile--clickable"
+		[ngClass]="{
+			'bx--tile--is-clicked': clicked
+		}"
+		tabindex="0"
+		(click)="onClick($event)"
+		(keydown)="onKeyDown($event)"
+		[href]="href"
+		[attr.aria-disabled]="disabled">
+		<ng-content></ng-content>
+	</a>`
+})
+export class ClickableTile {
+	/**
+	 * Sets the `href` attribute on the `ibm-clickable-tile` element.
+	 * @type {string}
+	 * @memberof ClickableTile
+	 */
+	@Input() href = "";
```

Comment:
This should be just `@Input() href: string;`.

With empty string on a page with url like "https://example.com/folder/#/stuff", it points to "https://example.com/folder/"
If it's not initialized, it won't be populated and won't point anywhere, which seems like the intent here.